go/consensus/tendermint/apps/beacon/api.go

```diff
@@ -1,25 +1,51 @@
 package beacon
 
-import "github.com/oasisprotocol/oasis-core/go/consensus/tendermint/api"
+import (
+	"github.com/tendermint/tendermint/abci/types"
+
+	beacon "github.com/oasisprotocol/oasis-core/go/beacon/api"
+	"github.com/oasisprotocol/oasis-core/go/consensus/api/transaction"
+	"github.com/oasisprotocol/oasis-core/go/consensus/tendermint/api"
+	beaconState "github.com/oasisprotocol/oasis-core/go/consensus/tendermint/apps/beacon/state"
+	genesis "github.com/oasisprotocol/oasis-core/go/genesis/api"
+)
 
 const (
 	// AppID is the unique application identifier.
-	AppID uint8 = 0x04
+	AppID uint8 = 0x08
```

Comment:
This should probably not change as the state prefixes are still `0x30`/`0x40`?. Also see discussion in https://github.com/oasisprotocol/oasis-core/pull/3423#discussion_r520704948. Either this or all the state key prefixes should change to `0x80` (in `beacon/state`).